src/Server/TCPHandler.cpp

```diff
@@ -1417,6 +1423,19 @@ void TCPHandler::receiveClusterNameAndSalt()
     readStringBinary(salt, *in, 32);
 }
 
+#if USE_SSL
+void TCPHandler::generateAndSendNonce()
+{
+    /// Contains lots of stuff (including time), so this should be enough for NONCE.
+    pcg64_fast rng(randomSeed());
+    UInt64 rand = rng();
+
+    nonce = encodeSHA256(&rand, sizeof(rand));
```

Comment:
`encodeSHA256` does not add any entropy, so why do we need it? What's the difference from, for example, `base64Encode`? Seems like it's equivalent to simply sending `rand`.

Reply:
Yes, this is useless in this case, and also will remove `USE_SSL` preprocessor check, which is great.